source/runtime.hpp

```diff
@@ -31,6 +31,29 @@ namespace reshade
 	class runtime abstract
 	{
 	public:
+		enum depth_buffer_retrieval_mode
+		{
+			POST_PROCESS = 0,
+			BEFORE_CLEARING_STAGE = 1,
+			AT_OM_STAGE = 2
+		};
+
+		enum depth_buffer_texture_type
+		{
+			BOTH = 0,
+			DEPTH_BUFFER = 1,
+			STENCIL_BUFFER = 2
+		};
+
+		enum class allowed_technique
```

Comment:
Just noticed that this is not used anywhere and should probably go.

Reply:
fixed